RELEASE.md

```diff
@@ -0,0 +1,15 @@
+# Releasing SlateDB
+
+SlateDB releases are published using a Github release action defined in `.github/workflows/release.yml`. To create a new release:
+
+1. Go to the [release action page](https://github.com/slatedb/slatedb/actions/workflows/pr.yaml)
```

Comment:
is the link correct? shouldn't it be `release.yaml`?

Reply:
Ah good catch. Fixed: e3c39cc